docs/typings-examples.md

```diff
@@ -0,0 +1,86 @@
+# Examples
+Here are some examples based on different shapes of the source module.
+
+## Exporting one function
+
+```ts
+declare function domready (...): any;
+
+export = domready
+```
+https://github.com/typed-typings/npm-domready
+
+## Export function with overloads
+```ts
+declare function xtend <A> (a: A): A;
+declare function xtend <A, B> (a: A, b: B): A & B;
+export = extend;
+```
+https://github.com/typed-typings/npm-xtend
+
+## Exporting function + utility
```

Comment:
Oh, sorry, just either Tape or Globby. What if there was a list instead? So the format becomes:

~~~md
## <title>

<description>

```ts
<code>
```

* <library>
* ...
~~~